### PR TITLE
Feature i18n support timesheets

### PR DESCRIPTION
**Pull request by bellyhold**
Timesheets should be complete. Ready for review.

### CODE REVIEW

File: resources/language/en-US.ini [hunk 1]
```diff
@@ -283,7 +294,6 @@ links.all_todos = "All To-Dos"
 links.create_idea_board = "<i class='fa fa-plus'></i> Create New Board"
 links.edit_idea_board = "<i class='fa fa-edit'></i> Edit Board"
 links.add_more = "<span class='iconfa iconfa-plus'></span> Add More"
-links.delete_board = "<i class='fa fa-trash'></i> Delete Board"
```
Comment (marcelfolaron) [inline resources/language/en-US.ini]: Why did you remove that? Was that a duplicate?

File: resources/language/en-US.ini [hunk 2]
```diff
@@ -557,11 +593,10 @@ email_notifications.canvas_item_update_cta = "Click here to see it"
 
 email_notifications.canvas_board_item_created = "A new hypothesis was added to one of your canvases"
 email_notifications.canvas_item_created_message = "%1$s updated the item '%2$s'"
-email_notifications.canvas_item_update_cta = "Click here to see it"
```
Comment (marcelfolaron) [inline resources/language/en-US.ini]: Why remove these? (They say the same thing, but I wanted to keep them in case we wanted to change the content)

File: resources/language/en-US.ini (hunk 1 above)
Comment (bellyhold) [reply resources/language/en-US.ini]: Yeah. Duplicate.

File: resources/language/en-US.ini (hunk 2 above)
Comment (bellyhold) [reply resources/language/en-US.ini]: This was a duplicate as well. I got some duplicates after merging the main feature branch back in.
